package.json

```diff
@@ -33,6 +33,7 @@
     "ethereumjs-util": "7.1.3",
     "ethers": "5.5.1",
     "faker": "5.5.3",
+    "figma-api": "1.9.2",
```

Comment:
this could be moved to dev dependencies but it isn't a big deal

Reply:
yes, totally agree